modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
Any use of abs_tol in a test is frowned upon (whether it's large or
small).  Try to remove it and see if you can get the test to work.

Derek

On Tue, May 5, 2015 at 6:34 PM Andy Wilkins <notifications@github.com>
wrote:

> In modules/tensor_mechanics/tests/mohr_coulomb/tests
> <https://github.com/idaholab/moose/pull/5030#discussion_r29720483>:
>
> > @@ -126,6 +133,13 @@
> >      rel_err = 1.0E-5
> >      abs_zero = 1.0E-3
> >    [../]
> > +  [./uni_axial1_small_strain]
> > +    type = 'CSVDiff'
> > +    input = 'uni_axial1_small_strain.i'
> > +    csvdiff = 'uni_axial1.csv'
> > +    rel_err = 1.0E-5
> > +    abs_zero = 1.0E-3
>
> I'll have a look tonight, @permcody <https://github.com/permcody> . Off
> the top of my head i'm not sure if nl_abs_tol=1E-4 is small or large.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5030/files#r29720483>.
>